setup.py

```diff
@@ -39,7 +44,10 @@ def _post_install(install_lib_dir):
     except sp.CalledProcessError as e:
         print("error:", e.output, file=sys.stderr)
     # IPython profile
-    user = os.environ["SUDO_USER"] if "SUDO_USER" in os.environ else os.environ["USER"]
+    if os.getenv('SUDO_USER'):
+        user = os.getenv("SUDO_USER")
+    else:
+        user = os.getenv("USER")
     home_dir = os.path.expanduser("~{}".format(user))
```

Comment:
The environment variable `USER` does not exist by default on Windows. I believe the lines 47 - 51 intend to capture the actual user name even if the user has `sudo`-oed in the current terminal or script. So in my opinion this should do it:
```
user = os.getenv('SUDO_USER')
user = os.path.expanduser("~{}".format(user)) if user else os.path.expanduser("~")
```
Please test on Windows and on Linux and in the latter case please also test with the `sudo` use case!